autogpt_platform/autogpt_libs/autogpt_libs/auth/config.py

```diff
@@ -1,14 +1,11 @@
 import os
 
-from dotenv import load_dotenv
-
-load_dotenv()
```

Comment:
how do we load env file then ?

Reply:
Loading the env is up to the top-level application and shouldn't be done by random library code *at import time*. This specific file was causing issues because it got imported and would then use the closest .env file which was `autogpt_platform/.env` instead of `autogpt_platform/backend/.env` even though it was being imported by `backend`.